packages/client/public/index.html

```diff
@@ -7,6 +7,7 @@
   <link rel='shortcut icon' href='%PUBLIC_URL%/favicon.ico' />
   <link rel='stylesheet' href='%PUBLIC_URL%/leaflet.css' />
   <link rel='stylesheet' href='%PUBLIC_URL%/leaflet.select/leaflet.control.select.css' />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/flatpickr@4.6.3/dist/flatpickr.min.css">
```

Comment:
we'll need stylesheets to be stored in the repo please, we won't be able to load them at run-time.